CorsixTH/Lua/dialogs/resizables/options.lua

```diff
@@ -237,12 +237,17 @@ function UIOptions:checkForAvailableLanguages()
   local langs, c = {}, 1
   for _, lang in pairs(app.strings.languages) do
     local font = app.strings:getFont(lang)
-    if app.gfx:hasLanguageFont(font) and app.strings.languages_english[lang] then
-      local eng_name = app.strings.languages_english[lang]
-      c = c + 1
+    local eng_name = app.strings.languages_english[lang]
+    c = c + 1
+    -- If freetype support and a unicode font setting are not present then
+    -- languages not supported by the builtin font are named in English and cannot be selected
+    if app.gfx:hasLanguageFont(font) then
       font = font and app.gfx:loadLanguageFont(font, app.gfx:loadSpriteTable("QData", "Font01V"))
-      langs[#langs + 1] = { text = lang, font = font,
+      langs[#langs + 1] = { text = lang,     name = lang, font = font, disabled = false,
```

Comment:
Why the odd spacing?

Reply:
It's a little easier to compare with the similar line below when they're aligned.